selectolax/lexbor.pxd

```diff
@@ -111,32 +111,32 @@ cdef extern from "lexbor/html/html.h" nogil:
         lxb_html_document_opt_t         opt
 
     ctypedef  enum lxb_html_document_ready_state_t:
-        LXB_HTML_DOCUMENT_READY_STATE_UNDEF       = 0x00
-        LXB_HTML_DOCUMENT_READY_STATE_LOADING     = 0x01
+        LXB_HTML_DOCUMENT_READY_STATE_UNDEF = 0x00
+        LXB_HTML_DOCUMENT_READY_STATE_LOADING = 0x01
         LXB_HTML_DOCUMENT_READY_STATE_INTERACTIVE = 0x02
-        LXB_HTML_DOCUMENT_READY_STATE_COMPLETE    = 0x03
+        LXB_HTML_DOCUMENT_READY_STATE_COMPLETE = 0x03
 
     ctypedef enum lxb_html_parser_state_t:
-        LXB_HTML_PARSER_STATE_BEGIN            = 0x00
-        LXB_HTML_PARSER_STATE_PROCESS          = 0x01
-        LXB_HTML_PARSER_STATE_END              = 0x02
+        LXB_HTML_PARSER_STATE_BEGIN = 0x00
+        LXB_HTML_PARSER_STATE_PROCESS = 0x01
+        LXB_HTML_PARSER_STATE_END = 0x02
         LXB_HTML_PARSER_STATE_FRAGMENT_PROCESS = 0x03
-        LXB_HTML_PARSER_STATE_ERROR            = 0x04
+        LXB_HTML_PARSER_STATE_ERROR = 0x04
 
     ctypedef enum lxb_dom_node_type_t:
-        LXB_DOM_NODE_TYPE_ELEMENT                = 0x01
-        LXB_DOM_NODE_TYPE_ATTRIBUTE              = 0x02
```

Comment:
Such formatting is intentional. Please avoid changing formatting everywhere. It makes it harder to review and keeps unrelated changes in the same commits.